lib/avromatic/model/types/decimal_type.rb

```diff
@@ -36,7 +36,7 @@ def coerce(input)
           when ::NilClass, ::BigDecimal
             input
           when ::Rational
-            input.to_d(precision)
+            input.to_d(scale)
```

Comment:
It has to be `precision`.

> The required `precision` parameter is used to determine the **number of significant digits** for the result.

It's not the number of decimal points.

Here is the example:

```ruby
require 'bigdecimal/util'

irb(main):073:0> r = 3001 / 3r
=> (3001/3)
irb(main):074:0> r.to_f
=> 1000.3333333333334
irb(main):075:0> r.to_d(2).to_f
=> 1000.0
irb(main):076:0> r.to_d(18).to_f
=> 1000.3333333333334
irb(main):077:0> r.to_d(6).to_f
=> 1000.33
```